src-built-in/components/toolbar/src/app.jsx

```diff
@@ -21,10 +21,12 @@ const Toolbar = () => {
 				{/* Workspace Management Menu */}
 				{/* App Menu */}
 			</div>
-			<div className="finsemble-toolbar-section center">
+			<ToolbarSection className="center" minWidth={115}>
+				<div className="divider"></div>
 				<FavoritesShell />
-			</div>
+			</ToolbarSection>
```

Comment:
I like `ToolbarSection`, but I don't like that we now have a mix of `div`'s and `ToolbarSection`'s. I would prefer we use `ToolbarSection`s for every child of `Toolbar`, rather than the mix.

Reply:
Yeah I agree, I was just asked to take care of the middle section. @brad should we use ToolbarSection for the rest? in this PR?

Reply:
@amaroks yeah let's make it consistent

Reply:
done, can you re review please